internal/querynode/data_sync_service.go

```diff
@@ -39,6 +39,8 @@ type dataSyncService struct {
 	metaReplica  ReplicaInterface
 	tSafeReplica TSafeReplicaInterface
 	msFactory    msgstream.Factory
+
+	closeCh chan struct{}
```

Comment:
maintain a `closeChan` here, but did nothing on it ?

Reply:
Removed